release-notes/index.dita

```diff
@@ -86,61 +86,139 @@
-          </li>
-          <li id="3792">Populate ancestor links during map pull step
+          <li id="3792">The mappull processing step was changed regarding links are generated with
+              <parmname>args.rellinks</parmname>. Starting in 3.7 <option>noparent</option> will not generate any
+            ancestor links and <option>nofamily</option> will not generate sibling, cousin, ancestor, or descendant
+            links. Prior to 3.7 <parmname>args.rellinks </parmname><option>all</option> did not actually include all
+            links. Now it will. If you used <option>all</option> and want to retain the links as they were generated
+            prior to 3.7, you must specify the Ant property  <codeph>include.rellinks="#default parent child sibling
+              friend next previous cousin descendant sample external other"</codeph>.
             <xref href="https://github.com/dita-ot/dita-ot/issues/3792" format="html" scope="external">#3792</xref>
           </li>
-          <li id="3794">Use modes instead of named templates in HTML5 stylesheets
-            <xref href="https://github.com/dita-ot/dita-ot/issues/3794" format="html" scope="external">#3794</xref>
-          </li>
-          <li id="3796">merge duplicate @keyscope values between <xmlelement>mapref</xmlelement> and <xmlelement
-            >map</xmlelement>
+          <li id="3796">Mapref processing was improved to remove any duplicate <xmlatt>keyscope</xmlatt> values. Prior
+            to 3.7 it was possible that <xmlelement>mapref</xmlelement> and <xmlelement>map</xmlelement> would
+            contribute the same <xmlatt>keyscope</xmlatt> value when the value was defined on both.
             <xref href="https://github.com/dita-ot/dita-ot/issues/3796" format="html" scope="external">#3796</xref>
           </li>
-          <li id="3803">Update to Gradle 7.2
-            <xref href="https://github.com/dita-ot/dita-ot/issues/3803" format="html" scope="external">#3803</xref>
-          </li>
-          <li id="3806">Add commonattributes mode in HTML5 to allow for easier extension
+          <li id="3806">A <codeph>commonattributes</codeph> mode was added to HTML5, PDF, and XHTML plugins to allow for
+            easier extension. This is a backwards compatible change, however, existing plug-ins should be changed to use
+            the new <codeph>commonattributes</codeph> mode.
             <xref href="https://github.com/dita-ot/dita-ot/issues/3806" format="html" scope="external">#3806</xref>
-          </li>
-          <li id="3807">Remove PDF2 AXF and RenderX in favour of separate repo
+            <fig id="fig_jq5_h5h_qrb">
+              <title>Pre-3.7</title>
+              <codeblock id="codeblock_tcz_g5h_qrb">&lt;xsl:template name="commonattributes">
+  &lt;!-- whole copy of commonattributes named template with customizations -->
+&lt;/xsl:template></codeblock>
+            </fig><fig id="fig_uwr_j5h_qrb">
+              <title>3.7</title>
```

Comment:
We should probably spell this out for clarity, and add XML syntax highlighting to the code blocks